src/eventgrid/azext_eventgrid/custom.py

```diff
@@ -1242,9 +1497,12 @@ def _update_event_subscription_internal(
         labels=event_subscription_labels,
         retry_policy=retry_policy,
         dead_letter_destination=deadletter_destination,
-        event_delivery_schema=event_delivery_schema
-    )
+        event_delivery_schema=event_delivery_schema,
+        delivery_with_resource_identity=delivery_with_resource_identity,
+        dead_letter_with_resource_identity=deadletter_with_resource_identity)
 
+    print('====> params')
```

Comment:
`print` is not recommended in azure cli. Please use `logger`.

Reply:
Thanks. was using for debugging and forgot to remove.